src/include/common/assert.h

```diff
@@ -14,12 +14,14 @@ namespace common {
     // LCOV_EXCL_STOP
 }
 
-#if defined(KUZU_RUNTIME_CHECKS) || !defined(NDEBUG)
-#define RUNTIME_CHECK(code) code
-#define KU_ASSERT(condition)                                                                       \
+#define KU_ASSERT_UNCONDITIONAL(condition)                                                         \
```

Comment:
Open to a better naming for this assert that is not optionally disabled with make flags.